cpp/README.md

```diff
@@ -1,158 +1,62 @@
 # C++ Demos
 
-- [Overview](#overview)
-- [Building and Running the Demos on Linux and macOS](#building-and-running-the-demos-on-linux-and-macos)
-- [Building and Running the Demos on Windows](#building-and-running-the-demos-on-windows)
+- [C++ Demos](#c-demos)
+  - [Overview](#overview)
+  - [macOS Prerequisites](#macos-prerequisites)
+  - [Ubuntu Prerequisites](#ubuntu-prerequisites)
```

Comment:
It sounds like we're dropping all other Linux distributions. You should be able to build the demos on RHEL!

Reply:
Yes I still need to add RHEL